### PR TITLE
Scheduler: fix code style issues for pkg/scheduler

### PR DESCRIPTION
**Pull request by SataQiu**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
scheduler: fix code style issues for `pkg/scheduler`:
- uniform the receiver name as `sched`
- make the comments consistent with the function name

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/runtime/framework.go [hunk 1]
```diff
@@ -602,7 +602,7 @@ func updatePluginList(pluginList interface{}, pluginSet config.PluginSet, plugin
 	return nil
 }
 
-// EnqueuePlugins returns the registered enqueue plugins.
+// PreEnqueuePlugins returns the registered enqueue plugins.
```
Comment (AxeZhan) [inline pkg/scheduler/framework/runtime/framework.go:605]:
```suggestion
// PreEnqueuePlugins returns the registered preEnqueue plugins.
```

**Comment by sanposhiho** [comment]
Sorry about saying something troublesome, but please `/assign` an approver after you've got `/lgtm` from someone in a basic case. We want to make each PR go through two reviews basically. I'll check this PR after `/lgtm`-ed.

File: pkg/scheduler/framework/runtime/framework.go (hunk 1 above)
Comment (SataQiu) [reply pkg/scheduler/framework/runtime/framework.go]: Thanks @AxeZhan Done!

**Comment by AxeZhan** [comment]
well, other things lgtm.
/lgtm

File: pkg/scheduler/apis/config/validation/validation_pluginargs.go [hunk 2]
```diff
@@ -271,7 +271,7 @@ func ValidateVolumeBindingArgs(path *field.Path, args *config.VolumeBindingArgs)
 	})
 }
 
-// ValidateVolumeBindingArgs validates that VolumeBindingArgs with scheduler features.
+// ValidateVolumeBindingArgsWithOptions validates that VolumeBindingArgs with scheduler features.
```
Comment (sanposhiho) [inline pkg/scheduler/apis/config/validation/validation_pluginargs.go:274]:
```suggestion
// ValidateVolumeBindingArgsWithOptions validates that VolumeBindingArgs and VolumeBindingArgsValidationOptions with scheduler features.
```

File: pkg/scheduler/testing/wrappers.go [hunk 3]
```diff
@@ -930,12 +930,12 @@ type PodSchedulingWrapper struct {
...
 }
 
-// MakePodSchedulingContext creates a PodSchedulingContext wrapper.
+// MakePodSchedulingContexts creates a PodSchedulingContext wrapper.
 func MakePodSchedulingContexts() *PodSchedulingWrapper {
 	return &PodSchedulingWrapper{resourcev1alpha2.PodSchedulingContext{}}
 }
 
-// FromPodSchedulingContext creates a PodSchedulingContext wrapper from some existing object.
+// FromPodSchedulingContexts creates a PodSchedulingContext wrapper from some existing object.
```
Comment (sanposhiho) [inline pkg/scheduler/testing/wrappers.go:938]:
```suggestion
// FromPodSchedulingContexts creates a PodSchedulingContext wrapper from an existing PodSchedulingContext.
```

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

Thanks

**Pull request merged**